its/src/test/java/com/sonar/maven/it/suite/JavaTest.java

```diff
@@ -120,6 +122,30 @@ void setJavaVersionProperties() throws IOException {
       entry("sonar.java.target", "1.8"));
   }
 
+  @Test
+  void when_java_home_exists_it_is_used_as_default_sonar_java_jdkHome_value() throws IOException {
+    //verify that property java home exists, it may not be pointing to a jdk
+    Assumptions.assumeTrue(System.getProperties().containsKey("java.home"));
+
```

Comment:
If this is always true, then we can remove the assumption.
```suggestion
```

Reply:
:+1: